bentoml/_internal/configuration/__init__.py

```diff
@@ -34,14 +38,14 @@ def expand_env_var(env_var):
 # the BentoML module to be used when loading and using a saved BentoService bundle.
 # This is useful when using customized BentoML fork/branch or when working with
 # development branches of BentoML
-BENTOML_VERSION = __version__
+BENTOML_VERSION: str = importlib_metadata.version(__name__)
```

Comment:
I assume this reads the PKG-INFO file from the package sdist?  what does it return in a dev environment?   I'm confused how this would interact with versioneer

Reply:
I wrote this up somewhere and then promptly forgot, probably---in a dev environment it evaluates the setup.py or pyproject.toml versions, it seems like.

So it works as "expected" giving 0.13.x+g<whatever>, at least for me right now with the poetry venv.